src/docfx/build/link/LinkResolver.cs

```diff
@@ -350,4 +392,23 @@
 
         _contentValidator.ValidateLink(file, node);
     }
+
+    private bool TryResolveAbsolutePathAsRelativePath(string absolutePath, FilePath referencingFile, bool lookupFallbackCommits, bool contentFallback, out FilePath? relativePath)
+    {
+        var absolutePathReplaceSlash = absolutePath.TrimStart(new char[] { '\\', '/' }).Replace('\\', '/');
+        var basePath = absolutePathReplaceSlash.Split('/')[0];
+        if (string.Equals(_config.BasePath, basePath, StringComparison.Ordinal))
+        {
+            var path = TryResolveRelativePath(
```

Comment:
Seems this method may return `null`, so when the returned `path` is null we still return true?